src/redis_lock/__init__.py

```diff
@@ -156,7 +157,9 @@ def __init__(self, redis_client, name, expire=None, id=None, auto_renewal=False)
         self._held = id is not None
         self._name = 'lock:'+name
         self._signal = 'lock-signal:'+name
-        self._lock_renewal_interval = expire*2/3 if auto_renewal else None
+        self._lock_renewal_interval = (float(expire)*2/3
+                                       if auto_renewal
+                                       else None)
```

Comment:
So if expire is `1` then `1*2/3` would had been `0`. Shouldn't we just do a `from __future__ import division` instead? Just in case we do more "maths" in other places.

But if it was 0, how did it work anyway for other interpreters?! It run ran often enough (spinloop) to not get gc-ed?